app-arch/tar/tar-1.29.recipe

```diff
@@ -26,17 +26,33 @@ PROVIDES="
 REQUIRES="
 	haiku
 	"
+
 BUILD_REQUIRES="
 	haiku_devel
 	"
 BUILD_PREREQUIRES="
 	cmd:automake
 	cmd:autoreconf
+	cmd:find
 	cmd:gcc
 	cmd:ld
 	cmd:make
 	cmd:sed
 	"
+if [ "$recipeAction" = TEST ]; then
+	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
```

Comment:
OK. Shall I drop the conditional definition and put **`cmd:compress`**, **`cmd:gzip`** and **`cmd:bzip2`** in BUILD_PREREQUIRES and drop the rest: **`cmd:lzip`**, **`cmd:lzma`** and **`cmd:xz`**.
Or is it OK to keep the the small condition on "$effectiveTargetArchitecture" != x86_gcc2 ?